packages/backend/src/routers/projectRouter.ts

```diff
@@ -377,3 +377,76 @@ projectRouter.get(
         }
     },
 );
+
+projectRouter.get('/projectAccess', isAuthenticated, async (req, res, next) => {
```

Comment:
For consistency can we use hyphens for all urls like `/password-reset` or `/invite-links`